src/main/java/seedu/address/model/house/HousingType.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.model.house;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+
+/**
+ * Represents a House's housing type.
+ * Guarantees: immutable; is valid as declared in {@link #isValidLevel(String)}
+ */
+public class HousingType {
```

Comment:
The naming of this need to follow the new class diagram, which is "InterestedHousingType"